### ISSUE TITLE
[FEATURE REQUEST] isValidPhone(type) instead of isValidPhone()

### ISSUE THREAD

**Issue by silvioprog** [brazilian-utils/brazilian-utils#97]
Hi.

It would be nice to allow to validade a phone number by its type, for example:

```js
const test = isValidPhone(); // Validates both phone and mobile phone number
const test = isValidPhone(false); // Validates only phone number
const test = isValidPhone(true); // Validates only mobile phone number
```

or maybe providing separated functions, like `isValidPhone()` and `isValidMobilePhone()`.

cheers

**Comment by hyanmandian** [comment]
Hello @silvioprog , nice idea! I think that we should keep `isValidPhone` validating both` mobile phone` and `landline phone` and create another 2 utilities:` isValidLandlinePhone `and` isValidMobilePhone` to validate each type separatly. In that way, we don't break our current API: D

What do you think about that?

**Comment by silvioprog** [comment]
Hello @hyanmandian, `isValidLandlinePhone()` and `isValidMobilePhone()` sounds very good, I totally agree with them. 😃 

I'm excited with `brazillian-utils`, this is really a awesome project. Thank you for maintaining it! 😄

**Comment by hyanmandian** [comment]
Thanks! I don't have time right now to implement these utilities (maybe next week), but you can contribute with our project if you feel comfortable with that.

**Comment by silvioprog** [comment]
@hyanmandian Great! So, i'm going to fork this repo and try to implement it to send as PR ...

**Comment by hyanmandian** [comment]
closed by #98
